GEOSwift/GEOS.swift

```diff
@@ -277,7 +277,7 @@ public struct GeometriesCollection<T: Geometry>: Sequence {
     }
 }
 
-public struct Coordinate {
+public struct Coordinate: Hashable {
```

Comment:
This seemed like an easy improvement for everyone, but it also makes writing tests easier.